docs/IE_DG/network_state_intro.md

```diff
@@ -0,0 +1,222 @@
+Introduction to OpenVINO state API {#openvino_docs_IE_DG_network_state_intro}
+==============================
+
+This section provides description how to work with networks with state in OpenVINO: how such networks 
+are represented in IR and Ngraph and operations with state can be done. Section provides small examples 
+of network with state and how it can be inferred.
+
+## What is network with state
+
+ Several use cases require processing of data sequences. When length of sequence is known and small enough, 
+ we can process it with RNN like networks which contains cycle inside. But in some cases like online speech recognition of time series 
+ forecasting length of data sequence is unknown. Then data can be divided in small portions and processed step by step. But dependency 
+ between data portions should be addressed. For that networks saved some data between inferences - state. When one dependent sequence is over,
+ state should be reset to initial value and new sequence can be started.
+ 
+ Several frameworks have special API for states in networks. For example, Keras have special option for RNNs `stateful` that turn on saving state 
+ between inferences. Kaldi contains special specifier `Offset` to define time offset in network. 
+ 
+ OpenVINO contains special API to simplify work with networks with states too. State will be automatically saved between inferences 
+ and there is a way to reset state when needed. Also user can read state or set it to some new value between inferences.
+ 
+## OpenVINO state representation
+
+ OpenVINO contains special abstraction Variable to represent state in network. There are 2 operations to work with state: 
+* `Assign` to save value in state
+* `ReadValue` to read value saved on previous iteration.
+
+More details on these operations you can find in [specification](../ops/opset3.md).
+
+## Examples of representation of network with states
+
+To get model with states ready for inference you can convert model from another framework with ModelOptimizer to IR or create it with Ngraph builder. 
```

Comment:
"Ngraph builder"? Maybe it's better to say "to create ngraph function" and add a link to https://docs.openvinotoolkit.org/latest/openvino_docs_nGraph_DG_build_function.html

Reply:
fixed